manifests/v1beta1/components/postgres/postgres.yaml

```
@@ -0,0 +1,42 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: katib-postgres
+  namespace: kubeflow
+  labels:
+    katib.kubeflow.org/component: postgres
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      katib.kubeflow.org/component: postgres
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        katib.kubeflow.org/component: postgres
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      containers:
+        - name: katib-postgres
+          image: postgres:14.3-alpine
+          envFrom:
+            - secretRef:
+              name: katib-postgres-secrets
```

```suggestion
            - secretRef:
                name: katib-postgres-secrets
```

Since missing an indent in this field, all e2e tests failed.
https://github.com/kubeflow/katib/runs/7591453784?check_suite_focus=true#step:4:10287

---

Thanks for checking this carefully :+1: